completion/xeCJKfntef.cwl

```diff
@@ -0,0 +1,55 @@
+# xeCJKfntef package
+# Darcy Hu <hot123tea123@gmail.com> 2016
+#modified zepinglee 26 Jan 2021
+
+\CJKunderline{内容}
```

Comment:
Argument names in English?

Reply:
The arguments are taken from the original documentation of `xeCJK` in Chinese.

Reply:
Arg-names of other commands are already changed, not only by translating (e.g. Chinese to English) but also by renaming. For example, `\ctexset` is documented as `\ctexset{⟨键值列表⟩}` in `texdoc ctex`. But it is added as `\ctexset{keyvals}` in this PR, instead of either `\ctexset{键值列表%keyvals}` or `\ctexset{options%keyvals}`.

Reply:
I'll fix it to keep it consistent with the documentation.

Reply:
Though I don't think consistency with doc is a must ...